test/e2e/common/security_context.go

```diff
@@ -40,6 +43,152 @@ var _ = framework.KubeDescribe("Security Context", func() {
 		podClient = f.PodClient()
 	})
 
+	ginkgo.Context("when pod PID namespace is configurable [Feature:ShareProcessNamespace][NodeAlphaFeature:ShareProcessNamespace]", func() {
+		ginkgo.It("containers in pods using isolated PID namespaces should all receive PID 1", func() {
+			ginkgo.By("Create a pod with isolated PID namespaces.")
+			f.PodClient().CreateSync(&v1.Pod{
+				ObjectMeta: metav1.ObjectMeta{Name: "isolated-pid-ns-test-pod"},
+				Spec: v1.PodSpec{
+					Containers: []v1.Container{
+						{
+							Name:    "test-container-1",
+							Image:   imageutils.GetE2EImage(imageutils.BusyBox),
+							Command: []string{"/bin/top"},
+						},
+						{
+							Name:    "test-container-2",
+							Image:   imageutils.GetE2EImage(imageutils.BusyBox),
+							Command: []string{"/bin/sleep"},
+							Args:    []string{"10000"},
+						},
+					},
+				},
+			})
+
+			ginkgo.By("Check if both containers receive PID 1.")
+			pid1 := f.ExecCommandInContainer("isolated-pid-ns-test-pod", "test-container-1", "/bin/pidof", "top")
+			pid2 := f.ExecCommandInContainer("isolated-pid-ns-test-pod", "test-container-2", "/bin/pidof", "sleep")
+			if pid1 != "1" || pid2 != "1" {
+				framework.Failf("PIDs of different containers are not all 1: test-container-1=%v, test-container-2=%v", pid1, pid2)
+			}
+		})
+
+		ginkgo.It("processes in containers sharing a pod namespace should be able to see each other [Alpha]", func() {
```

Comment:
According to https://kubernetes.io/docs/reference/command-line-tools-reference/feature-gates/ this has been Beta since 1.12. Probably should remove the `[Alpha]` tag. This also means we cannot make this a conformance test unless it gets promoted to GA in 1.17.

Reply:
ack. I searched through the codebase and didn't see a `[Beta]` tag. Would that be unnecessary?

Reply:
yeah